### PR TITLE
Add check to sort dir when creating changefeed

### PR DESCRIPTION
**Pull request by amyangfei**
### What problem does this PR solve? 

Fix a bug reported by QA. If file sort is enabled and given `SortDir` is not a dir or writable, the changefeed will run with error

### What is changed and how it works?

Check given `SortDir` is a directory and writable

### Check List 

Tests 

 - Unit test
 - Integration test

### Release note

- No release note

### CODE REVIEW

File: cmd/client_changefeed.go [hunk 1]
```diff
@@ -243,6 +243,13 @@ func verifyChangefeedParamers(ctx context.Context, cmd *cobra.Command, isCreate
 		return nil, errors.Annotate(err, "can not load timezone, Please specify the time zone through environment variable `TZ` or command line parameters `--tz`")
 	}
 
+	if sortEngine == string(model.SortInFile) {
+		err = util.IsDirAndWritable(sortDir)
+		if err != nil {
+			return nil, err
+		}
+	}
```
Comment (overvenus) [inline cmd/client_changefeed.go:251]: I think it should be checked on CDC server.

Comment (amyangfei) [reply cmd/client_changefeed.go]: addressed, check on cdc server is more suitable.

**Pull request merged**